### PR TITLE
Added html for header

### LINKED ISSUE #11: html header
We need to create the layout of the header

### PR DESCRIPTION
**Pull request by AlekseyBelousov**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,7 +8,50 @@
...
+            <a href="#cashback">Cashback</a>
+          </li>
+          <!-- Item 5 -->
+          <li class="menu__item">
+            <a href="#clients">Clients</a>
+          </li>
+        </ul>
+        <div class="header__actions">
+          <!-- Language -->
+          <select class="header__lang" id="language">
```
Comment (Maiia2023) [inline index.html]: name for id is ok, but later you should always keep in mind that when Vitaly use id "lang" you should use "language"

File: index.html [hunk 2]
```diff
@@ -8,7 +8,50 @@
...
+            <option value="en">EN</option>
+            <option value="ru">RU</option>
+          </select>
+          <!-- Theme -->
+          <button class="header__theme" type="button" id="theme">
+            <img src="assets/icons/theme.svg" alt="Theme">
```
Comment (Maiia2023) [inline index.html]: Moon
Rename icon for moon
Later there will be icon sun (for light theme)

File: index.html [hunk 3]
```diff
@@ -8,7 +8,50 @@
...
+          </button>
+          <!-- Burger -->
+          <button class="header__burger" type="button" id="burger">
+
```
Comment (Maiia2023) [inline index.html]: add tag span

**Review by nagorapro — APPROVED** [review]
Good

**Review by Maiia2023 — APPROVED** [review]

**Pull request merged**